spec/acceptance/mod_passenger_spec.rb

```diff
@@ -171,10 +181,9 @@ class { 'apache::mod::passenger': }
       expected_one = [%r{Apache processes}, %r{Nginx processes}, %r{Passenger processes}]
       # passenger-memory-stats output on newer Debian/Ubuntu verions do not contain
       # these two lines
-      unless (fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemrelease') == '14.04') ||
-             (fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemrelease') == '16.04') ||
+      unless (fact('operatingsystem') == 'Ubuntu' && ['14.04', '16.04', '18.04'].include?(fact('operatingsystemrelease'))) ||
```

Comment:
Isn't this basically every supported Ubuntu version? I'd even thing Debian 9 would fail here so I think this whole block can be removed if it's no longer included.

Reply:
fixed